Sources/VitaminUIKit/Components/Badge/VitaminBadge.swift

```diff
@@ -208,7 +296,7 @@ public extension UIView {
     // Computes the badge frame relative to this view frame
     private func getBadgeFrame(badge: VitaminBadge) -> CGRect {
         CGRect(
-            x: self.frame.width - badge.offset,
+            x: self.frame.width - badge.theoricWidth / 2,
             y: badge.offset - badge.theoricHeight,
             width: badge.theoricWidth,
             height: badge.theoricHeight)
```

Comment:
If that's is OK for you, I would prefer to separate that in another issue/PR.
To be honest, I made a quick review of showcase, and noticed that the UIKit badges did not work properly, I began to fix it, and then noticed there was a gap with SwiftUI, and decided to implement size.

But since the very start was a bugfix, I would like to be able to merge this one relatively quick ;)

Reply:
Yes, it's good for me. (It was in my first comment 😉)